react-common/components/profile/Profile.tsx

```diff
@@ -44,5 +46,24 @@ export const Profile = (props: ProfileProps) => {
             userState={userBadges}
             onBadgeClick={onBadgeClick}
         />}
+
+        {profileSmall &&
+            <div className="profile-info-container">
+                <p className="profile-info">
+                    {lf("Now that you're logged in, your projects will be automatically saved to the cloud so you can access them from any device! ")}
+                    {lf("Learn more at ")}
+                    <a href="https://arcade.makecode.com/identity/cloud-sync" target="_blank" rel="noopener noreferrer" tabIndex={0}>{lf("Cloud Sync ")}</a>
+                    {lf("or ")}
```

Comment:
we shouldn't be pointing at target specific docs pages; either need to go under makecode.com/docs or be under common-docs with a relative link here so they get mirrored in each target.

Reply:
Is this to avoid CORS issues?

Reply:
It's not a cors issue, it's that targets should be relatively independent and should not point at each other without explicitly meaning to do so. few reasons;

* docs have common macro replacements that e.g. insert the target into the documentation which could be confusing
* school districts might choose to only allow a single target (maybe they really don't like the idea of multiplayer or making games, but are fine with programming micro:bits) so this page wouldn't be available
* if they get to the search page and poke around by searching or using the side menu to find more info they'll be on a page that doesn't relate to what they're learning (e.g. incorrect api references, different simulator, screenshots of arcade features that could be in different places not implemented in micro:bit, etc)